artiq/frontend/artiq_master.py

```diff
@@ -94,6 +96,12 @@ def ccb_issue(service, *args, **kwargs):
     scheduler.start()
     atexit_register_coroutine(scheduler.stop)
 
+    class MasterConfig:
+        def get_name(self):
+            return args.name
```

Comment:
For the purposes of limiting the complexity of the ``main`` function, which is getting quite heavy already, I would put the class definition outside.

Reply:
The name can be easily passed to the constructor.

Reply:
OK